internal/core/thirdparty/tantivy/tantivy-binding/Cargo.toml

```diff
@@ -15,6 +15,7 @@ lindera-cc-cedict = ["lindera/cc-cedict"]
 
 [dependencies]
 tantivy = { git = "https://github.com/zilliztech/tantivy.git" }
+tantivy-5 = { package = "tantivy", git = "https://github.com/quickwit-oss/tantivy.git", tag = "0.21.1" }
```

Comment:
should we use  [0.21.1-fix3](https://github.com/milvus-io/tantivy/releases/tag/0.21.1-fix3) in milvus-io/tantivy?

Reply:
Fine